### DISCUSSION TITLE
Errors in self signed Mosquitto doc and how to fix them

### DISCUSSION THREAD

**Discussion by lhurt** [arendst/Tasmota#16603] [Show and tell]
There's no way to file an issue because it has to stick to the bug template to not get closed automatically. Therefore I try to share my finding here hoping that someone corrects the documentation.

The page on [github.io documenting a self signed Mosquitto installation](https://tasmota.github.io/docs/Self-signed-Mosquitto/) has some errors.

It took me some time to successfully build an TLS version with full CA validation with my local CA. In order for other guys to save time the following changes have to be made.

[In section 2.2 Advanced: Using Full certificate validation](https://tasmota.github.io/docs/Self-signed-Mosquitto/#22-advanced-using-full-certificate-validation)

_Wrong macro identifier_
`#define INCLUDE_LOCAL_CA_CERT`

_Correct identifier_
`#define INCLUDE_LOCAL_CERT`

--

_Wrong path_
Copy or move the files created in step 1.6 above (local_ca_data.h and local_ca_descriptor.h) to the `$TASMOTAROOT/tasmota`

_Correct path_
Copy or move the files created in step 1.6 above (local_ca_data.h and local_ca_descriptor.h) to the `$TASMOTAROOT/tasmota/include`

--

_Wrong (filename typo)_
If desired, the two built-in certificates for Let's Encrypt and Amazon AWS may be omitted from the build by defining these macros in `user_config_overrirde.h`

_Correct_
If desired, the two built-in certificates for Let's Encrypt and Amazon AWS may be omitted from the build by defining these macros in `user_config_override.h`

**Comment by sfromis** [comment]
On the docs pages, there is an edit link top right, where users are welcome to suggest changes if/when spotting something in need of improvements.

**Reply by lhurt** [reply]
Thanks for the information. Created a pull request for the change.
